_source/reference/events.md

```diff
@@ -37,3 +37,5 @@ Turbo emits events that allow you to track the navigation lifecycle and respond
 * `turbo:frame-render` fires right after `<turbo-frame>` element renders its view. The specific `<turbo-frame>` element is the event target. Access the `FetchResponse` object with `event.detail.fetchResponse` property.
 
 * `turbo:frame-load` fires when `<turbo-frame>` element is navigated and finishes loading (fires after `turbo:frame-render`). The specific `<turbo-frame>` element is the event target.
+
+* `turbo:fetch-request-error` fires when a form or frame fetch request fails due to network errors. This event fires on the respective element (turbo-frame or form element) which triggers it and can be accessed with `event.target` property.
```

Comment:
Will this also fire during a `Visit` from an `<a>` click? Will the `document.documentElement` be the target?

Reply:
No, but perhaps it should? I think that would mean emitting this even here https://github.com/hotwired/turbo/blob/256418fee0178ee483d82cd9bb579bd5df5a151f/src/core/drive/visit.ts#L382

Reply:
I'm in favor of that change!

Reply:
I've opened https://github.com/hotwired/turbo/pull/685 to add that behavior.

It also makes the `turbo:fetch-request-error` event cancelable, which is also worth noting here.

Reply:
What happens when the event is canceled? Or is it enough to just say “This event can be canceled”.